### PR TITLE
docs: remove references to create-daml-app repo

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Hopefully that's all of them. Thanks to Alexander Bernauer for reporting on [the forum].

[the forum]: https://discuss.daml.com/t/type-script-support-dropped-with-sdk-1-9-0/2054/6?u=gary_verhaegen

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: docs/configs/html/conf.py [hunk 1]
```diff
@@ -175,7 +175,7 @@
 .. _installer: https://github.com/digital-asset/daml/releases/download/v{release}/daml-sdk-{release}-windows.exe
 .. _protobufs: https://github.com/digital-asset/daml/releases/download/v{release}/protobufs-{release}.zip
 .. _api-test-tool: https://repo1.maven.org/maven2/com/daml/ledger-api-test-tool/{release}/ledger-api-test-tool-{release}.jar
-.. _http-json: https://github.com/digital-asset/daml/releases/download/v{release}/http-json-{release}.jar
```
Comment (garyverhaegen-da) [inline docs/configs/html/conf.py:178]: `http-json` doesn't seem to be used anywhere. That's probably why it wasn't an issue that it was missing on the PDF config.

File: docs/source/app-dev/app-arch.rst [hunk 2]
```diff
@@ -109,22 +108,24 @@ form of the required tokens is described in the :ref:`Authorization <authorizati
...
-  #. Make a small change to your Daml data model
-  #. Optionally test your Daml code and with :ref:`scenarios <testing-using-scenarios>`
+1. The integrated VSCode IDE (``daml studio``) runs your Scripts on any change to your Daml models. See :ref:`Daml scenarios <testing-using-scenarios>`.
+#. ``daml start`` will build all of your Daml code, generate the JavaScript bindings, and start the required "backend" processes (sandbox and HTTP JSON API). It will also allow you to press ``r`` (followed by Enter on Windows) to rebuild your code, regenerate the JavaScript bindings and upload the new code to the running ledger.
+#. ``npm start`` will watch your JavaScript source files for change and recompile them immediately when they are saved.
+
+Together, these features can provide you with very tight feedback loops while developing your Daml application, all the way from your Daml contracts up to your web UI. A typical Daml developer workflow is to
+
+  1. Make a small change to your Daml data model
+  #. Optionally test your Daml code with :ref:`scenarios <testing-using-scenarios>`
```
Comment (garyverhaegen-da) [inline docs/source/app-dev/app-arch.rst:120]: Not as part of this PR, but should this page be updated to Scripts? Any reason why we still mention scenarios at all, or is that just an oversight?

Comment (cocreature) [reply docs/source/app-dev/app-arch.rst:120]: should be updated yeah, I think I just missed that

File: docs/source/tools/assistant.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ Daml Assistant (``daml``)
 ``daml`` is a command-line tool that does a lot of useful things related to the SDK. Using ``daml``, you can:
 
 - Create new Daml projects: ``daml new <path to create project in>``
-- Create a new project based on `create-daml-app <https://github.com/digital-asset/create-daml-app>`_: ``daml create-daml-app <path to create project in>``
+- Create a new project based on the create-daml-app_ template: ``daml new --template=create-daml-app <path to create project in>``
```
Comment (cocreature) [inline docs/source/tools/assistant.rst:10]: is that link actually useful? why do we want to link to our repo here?

File: language-support/ts/daml-ledger/README.md [hunk 4]
```diff
@@ -10,8 +10,10 @@ Comprehensive documentation for `@daml/ledger` can be found
 
 ## Usage
 
-The best way to get you started quickly is to start with the `create-daml-app`
-template:
+The best way to get you started quickly is to use the [`create-daml-app` template]:
+
+[`create-daml-app` template]: https://github.com/digital-asset/daml/tree/v0.0.0-SDKVERSION/templates/create-daml-app
```
Comment (cocreature) [inline language-support/ts/daml-ledger/README.md:15]: similar question here, I’m not sure linking to the repo is helpful.

**Review by cocreature — APPROVED** [review]
lgmt, I’d just remove the links to the repo completely.

File: docs/source/tools/assistant.rst (hunk 3 above)
Comment (garyverhaegen-da) [reply docs/source/tools/assistant.rst:10]: There used to be a link, so I thought I'd preserve that. The main advantage I can think of is that it lets people see what the project would look like without creating it. Agree that's minimal, happy to remove it.

Comment (cocreature) [reply docs/source/tools/assistant.rst:10]: With all the `.template` files in there, I’d say it’s probably more confusing than helpful so I’d just remove it.

**Pull request merged**